bot/common/tasks/weekly_contributions.py

```diff
@@ -40,74 +44,146 @@ async def save_weekly_contribution_reports():
         logger.info(f"done saving report {path}")
 
 
-async def send_weekly_contribution_reports(bot):
+async def send_weekly_contribution_reports(bot: Bot):
     # retrieve the reporting channel
-    govrn_guild_id = constants.Bot.govrn_guild_id
-    govrn_guild = await get_guild_by_id(govrn_guild_id)
-    # TODO
-    report_channel_id = govrn_guild.get("fields").get("report_channel")
-
-    # if none is specified, log message and return
-    if report_channel_id is None or report_channel_id == "":
-        logger.info(f"report_channel is not specified for guild {govrn_guild_id}")
-        return
+    govrn_guild_discord_id = constants.Bot.govrn_guild_id
+    warn_msg = None
+    if govrn_guild_discord_id is None:
+        warn_msg = "No discord id specified for govrn"
+
+    govrn_guild = await get_guild_by_discord_id(govrn_guild_discord_id)
+    if govrn_guild is None:
+        warn_msg = f"No guild found for govrn discord id {govrn_guild_discord_id}"
+
+    # default reporting channel will be the reporting channel specified for
+    # govrn (that is, whatever is specified as the govrn discord id in the .env)
+    # all guilds which are registered but do not have a specified reporting
+    # channel will default to reporting in the specified govrn channel
+    default_reporting_channel_id = govrn_guild.get("contribution_reporting_channel")
+    if default_reporting_channel_id is None or default_reporting_channel_id == "":
```

Comment:
A simpler way to write the is would be !default_reporting_channel_id since both values are falsey